src/test/lib/DataTest.cpp

```diff
@@ -2960,6 +2960,116 @@ QuicTestStreamAbortConnFlowControl(
     TEST_TRUE(Context.ClientStreamShutdownComplete.WaitTimeout(TestWaitTimeout));
 }
 
+struct StreamReliableReset {
```

Comment:
Go ahead and put this inside the function below.

Reply:
Sounds good.